drivers/hmis/app/graphql/types/hmis_schema/organization_input.rb

```diff
@@ -0,0 +1,14 @@
+module Types
+  class HmisSchema::OrganizationInput < BaseInputObject
+    description 'HMIS Organization input'
+
+    argument :organization_name, String, required: false
+    argument :victim_service_provider, Boolean, required: false
+    argument :description, String, required: false
+    argument :contact_information, String, required: false
+
+    def to_params
+      to_h
```

Comment:
`victim_service_provider` needs to be mapped to 0, 1, or 99. I'm getting an error when i try to pass in `null` for the `victimServiceProvider` input

Reply:
The spec says that it's just a simple boolean, so that's how I implemented it. It sounds like it's actually a 1.7 field, so I changed to use `yes_no_missing_argument`. Hopefully that fixes it for you.

Reply:
Thanks - it just needs a yes_no_missing resolver to return the right thing, and then it should be good to go. Right now it always returns `true`.

Reply:
Fixed